filter_plugins/ceph-ansible.py

```diff
@@ -0,0 +1,13 @@
+from ansible import errors
+
+def only_one_true(a, *args):
+    params = list(args)
+    params.append(a)
+    it = iter(params)
+    return any(it) and not any(it)
+
+class FilterModule(object):
+    def filters(self):
+        return {
+            'only_one_true': only_one_true
```

Comment:
It cannot. The issue here is that we need to make sure that there is a True in the list, and only a _single_ true. What you purpose would ensure there is _a_ scenario set, but not a _single_ scenario set.